docs/guide/http_cache.md

```diff
@@ -387,351 +407,75 @@ $response = new Response();
-            set req.http.accept = "application/vnd.fos.user-context-hash";
-            set req.url = "/_fos_user_context_hash";
-
-            // Force the lookup, the backend must tell how to cache/vary response containing the user hash
-            return (hash);
-        }
-    }
-
-    // Rebuild the original request which now has the hash.
-    if (req.restarts > 0
-        && req.http.accept == "application/vnd.fos.user-context-hash"
-    ) {
-        set req.url         = req.http.x-fos-original-url;
-        set req.http.accept = req.http.x-fos-original-accept;
-        set req.http.cookie = req.http.x-fos-original-cookie;
-        unset req.http.x-fos-original-url;
-        unset req.http.x-fos-original-accept;
-        unset req.http.x-fos-original-cookie;
-
-        // Force the lookup, the backend must tell not to cache or vary on the
-        // user hash to properly separate cached data.
-
-        return (hash);
-    }
-}
-
-sub vcl_deliver {
-    // On receiving the hash response, copy the hash header to the original
-    // request and restart.
-    if (req.restarts == 0
-        && resp.http.content-type ~ "application/vnd.fos.user-context-hash"
-    ) {
-        set req.http.x-user-hash = resp.http.x-user-hash;
-        return (restart);
-    }
-
-    // If we get here, this is a real response that gets sent to the client.
-    // Remove the vary on context user hash, this is nothing public. Keep all
-    // other vary headers.
-    set resp.http.Vary = regsub(resp.http.Vary, "(?i),? *x-user-hash *", "");
-    set resp.http.Vary = regsub(resp.http.Vary, "^, *", "");
-    if (resp.http.Vary == "") {
-        unset resp.http.Vary;
-    }
-
-    // Sanity check to prevent ever exposing the hash to a client.
-    unset resp.http.x-user-hash;
-    if (client.ip ~ debuggers) {
-        if (obj.hits > 0) {
-            set resp.http.X-Cache = "HIT";
-            set resp.http.X-Cache-Hits = obj.hits;
-        } else {
-            set resp.http.X-Cache = "MISS";
-        }
-    }
-}
-```
-
-##### New anonymous X-User-Hash
-
-The anonymous X-User-Hash is generated based on the *anonymous user*, *group* and *role*. The `38015b703d82206ebc01d17a39c727e5` hash that is provided in the code above will work only when these three variables are left unchanged. Once you change the default permissions and settings, the X-User-Hash will change and Varnish won't be able to effectively handle cache anymore.
-
-In that case you need to find out the new anonymous X-User-Hash and change the VCL accordingly, else Varnish will return a no-cache header.
+The anonymous `X-User-Hash` is generated based on the *anonymous user*, *group* and *role*. The `38015b703d82206ebc01d17a39c727e5` hash that is provided in the code above will work only when these three variables are left unchanged. Once you change the default permissions and settings, the `X-User-Hash` will change and Varnish won't be able to effectively handle cache anymore.
```

Comment:
1. *anonymous user*, *group* and *role* are variables so maybe: `anonymous user`, `group` and `role`
2. `38015b703d82206ebc01d17a39c727e5` that is provided in the Tip/link above will work. 
    OR add to this point small part of the code with this hash.